### ISSUE TITLE
Wait for process to stop before restarting

### ISSUE THREAD

**Issue by JTWalraven** [ErnestOrt/Trampoline#46]
As a user, I do not want a microservice to restart (using the restart button) before the process running it has stopped.

Depends on #37

**Comment by ErnestOrt** [comment]
@JTWalraven did you check #31 issue. It's quite related, and would be good if you can keep it in mind when implementing this feature. It would help futute development on #31.

**Comment by JTWalraven** [comment]
@ErnestOrt, yeah I did see that. I must have created this issue before I looked it over. I actually pushed a branch with a rough cut version that uses http port checking instead of attempting to use the process logs.

**Comment by JTWalraven** [comment]
Closing in favor of the other issue.
